.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,11 +6,17 @@
 <!--
   Title
   Please include a concise title that briefly describes the change.
-  Titles should be prefixed with the area of code that the change affects and they should be in the present simple tense. Examples:
+  Titles should follow https://www.conventionalcommits.org/.
+  They should also be in the present simple tense.
+
+  Examples:
   
-  dvm: adds a new function to compute voting rewards offchain
-  monitor bot: fixes broken link in liquidation log
-  voter dapp: adds countdown timer component to the header
+  feat(dvm): adds a new function to compute voting rewards offchain
```

Comment:
+100 I always google what the diff options are. 

Maybe we can point to this [guide](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional) that contains a list of the various options

Reply:
What about using [this link](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type) instead? Just because it's a direct link to the list and doesn't require any searching around the page.